client/src/commands/createSmartContractProjectCommand.ts

```diff
@@ -274,7 +274,7 @@ async function getSmartContractLanguageOptions(): Promise<string[]> {
     return parsedJson.contractLanguages;
 }
 
-async function openNewProject(openMethod: string, uri: vscode.Uri): Promise<void> {
+export async function openNewProject(openMethod: string, uri: vscode.Uri): Promise<void> {
```

Comment:
This shouldn't be exported if you want to use this function put it somewhere else